### PR TITLE
Move docs in-source

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,12 +1,21 @@
 name: Build
 
 on:
+  # Run on pushes to master and pushed tags, and on pull requests against master, but ignore the docs folder
   push:
     branches: [ master ]
     tags:
       - 'v*'
+    paths:
```
Comment (mdurrani808) [inline .github/workflows/build.yml:9]: is the intent here just to ignore the docs path and include everything else?

File: .github/workflows/photonvision-docs.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - '.github/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
```
Comment (mdurrani808) [inline .github/workflows/photonvision-docs.yml:22]: 3.9 is specified here but 3.11 is specified below

File: .github/workflows/photonvision-docs.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    - name: Install Python dependencies
+      working-directory: docs
+      run: |
+        pip install sphinx sphinx_rtd_theme sphinx-tabs sphinxext-opengraph doc8
+        pip install -r requirements.txt
+
+    - name: Check links
+      working-directory: docs
+      run: make linkcheck || true
```
Comment (mdurrani808) [inline .github/workflows/photonvision-docs.yml:37]: is this so link check doesnt stop the build? i believe the right thing here would be continue-on-error

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

File: .github/workflows/build.yml (hunk 1 above)
Comment (mcm001) [reply .github/workflows/build.yml:9]: Yeh

Comment (mdurrani808) [reply .github/workflows/build.yml:9]: paths-ignore is reccomended here then 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

File: .github/workflows/photonvision-docs.yml (hunk 3 above)
Comment (mcm001) [reply .github/workflows/photonvision-docs.yml:37]: oh yeah

File: .github/workflows/photonvision-docs.yml (hunk 2 above)
Comment (mcm001) [reply .github/workflows/photonvision-docs.yml:22]: yeah ez fix. also fixed it so we are no longer cloning the docs repo anywhere

**Review by mdurrani808 — APPROVED** [review]

**Pull request merged**